data-transform/src/main/java/com/linkedin/data/transform/filter/FilterConstants.java

```diff
@@ -23,6 +23,7 @@
   public static final String  WILDCARD = "$*";
   public static final String  START = "$start";
   public static final String  COUNT = "$count";
+  public static final int NUMBER_OF_ARRAY_ATTRIBUTES = 2; // start and count.
```

Comment:
Sorry for my last comment. It might be better:
```
public static final Set<String> ARRAY_ATTRIBUTES = new HashSet<>(Arrays.asList(START, COUNT));
```
And then use `ARRAY_ATTRIBUTES.size()` in the code